### PR TITLE
Task 516: Moved reactionNames vector from PostgreRepositories to Base

### LINKED ISSUE #516: Move Reaction names list to Base section
## Description:
_DataAccess/DataAccess.Postgre/PostgreRepositories.cpp_ contain _reactionNames_  that looks lonely. Help to this vector move at the Base part of project to others Primitives that are common for the solution.

## Acceptance Criteria:
Program behaviour stay without changes

### PR DESCRIPTION
**Pull request by KliKonNik**
Moved reactionNames vector from PostgreRepositories.cpp to Base\Models\Primitives.hpp.

### CODE REVIEW

**Review by memorisecodead — APPROVED** [review]

File: DataAccess/DataAccess.Postgre/PostgreRepositories.cpp [hunk 1]
```diff
@@ -434,10 +434,12 @@ Utility::ReactionMessageCodes MessagesRepository::updateMessageReactions(const M
 {
     using Utility::ReactionMessageCodes;
 
+    /*!!!
     /*
     * @todo move reactionNames to Base
-    */
+    * /
     const std::vector<std::string> reactionNames = { "likes", "dislikes", "fires", "cats", "smiles" };
+    !!!*/
```
Comment (slavexx) [inline DataAccess/DataAccess.Postgre/PostgreRepositories.cpp]: I think, it can be removed

**Review by slavexx — COMMENTED** [review]
Have a little nuance

File: DataAccess/DataAccess.Postgre/PostgreRepositories.cpp (hunk 1 above)
Comment (KliKonNik) [reply DataAccess/DataAccess.Postgre/PostgreRepositories.cpp]: Done

**Review by KanstantsinTaukach — APPROVED** [review]

**Review by slavexx — APPROVED** [review]

**Review by ivanboika — APPROVED** [review]

**Review by ivanboika — COMMENTED** [review]
That's not correct rebase, changes from dev are there

**Review by memorisecodead — APPROVED** [review]

**Review by Orange-hanter — APPROVED** [review]
Looks good. You could close it

**Comment by memorisecodead** [comment]
Now, resolve merge conflict

**Pull request merged**